python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "kaskada"
-version = "0.6.0-a.1"
+version = "0.6.0-a.2"
```

Comment:
There are two version numbers in this -- should update both.

Is there a reason we're bumping this?